include/seqan/basic/alphabet_residue.h

```diff
@@ -482,7 +482,7 @@ unknownValueImpl(Rna5 *)
  *
  * @signature typedef SimpleType<unsigned char, Iupac_> Iupac;
  *
- * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 19 in this order: 'U'=0, 'T', 'A',
+ * The ValueSize of <tt>Iupac</tt> is 16.  The nucleotides are enumerated from 0 to 15 in this order: 'U'=0, 'T', 'A',
```

Comment:
while you are at it, I think the order might be also wrong...

The table [here](https://github.com/seqan/seqan/blob/master/include/seqan/basic/alphabet_residue_tabs.h#L303) suggests e.g. that `=` is 0 not `U` as in the documentation. Or am I wrong?
```cpp
template <typename T>
char const TranslateTableCharToIupac_<T>::VALUE[256] =
{
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    //                                                                =
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,   0,  15,  15,
    //    A,   B,   C,   D,   E,   F,   G,   H,   I,   J,   K,   L,   M,   N,   O,
    15,   1,  14,   2,  13,  15,  15,   4,  11,  15,  15,  12,  15,   3,  15,  15,
    //    Q,   R,   S,   T,   U,   V,   W,   X,   Y,   Z
    15,  15,   5,   6,   8,   0,   7,   9,  15,  10,  15,  15,  15,  15,  15,  15,
    //    a,   b,   c,   d,   e,   f,   g,   h,   i,   j,   k,   l,   m,   n,   o,
    15,   1,  14,   2,  13,  15,  15,   4,  11,  15,  15,  12,  15,   3,  15,  15,
    //    q,   r,   s,   t,   u,   v,   w,   x,   y,   z
    15,  15,   5,   6,   8,   0,   7,   9,  15,  10,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,
    15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15,  15
};
```

Reply:
Good catch! I tested it and updated the documentation.